src/next-transpile-modules.js

```diff
@@ -111,41 +111,50 @@ const withTmInitializer = (modules = [], options = {}) => {
      * @returns {string}
      */
     const getPackageRootDirectory = (module) => {
-      let packageDirectory;
+      let packageLookupDirectory;
       let packageRootDirectory;
 
       try {
-        // Get the module path
-        packageDirectory = resolve(CWD, module);
-
-        if (!packageDirectory) {
-          throw new Error(
-            `next-transpile-modules - could not resolve module "${module}". Are you sure the name module you are trying to transpile is correct, and it has a "main" or an "exports" field?`
+        packageLookupDirectory = resolve(CWD, path.join(module, 'package.json'));
```

Comment:
this joining doesn't work for e.g. `next/head` etc.

```
next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "next/head", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "next/router", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "styled-jsx/server", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
```

Reply:
- why would you need to transpile any part of `next`?
- from the readme: _"there is currently no way to transpile only parts of a package, it's all or nothing"_, so you cannot pick a package subfolder to be transpiled anyway (`styled-jsx/server` is equivalent to `styled-jsx`)

Reply:
I don't want to.

```js
const withTM = require('next-transpile-modules')([
  'folio-common-components',
  'folio-common-utils',
]);
```

this is my config. why it triggers this module and its warnings are beyond me, I haven't dug into it

Reply:
Can you paste a debug log here? (cf. README)

Reply:
<details>
<summary>Details</summary>

```
info  - Using webpack 4. Reason: custom webpack configuration in next.config.js https://nextjs.org/docs/messages/webpack5
next-transpile-modules - trying to resolve the following modules:
  - folio-common-components
  - folio-common-utils
next-transpile-modules - the following paths will get transpiled:
  - /Users/simen/repos/monorepo/packages/folio-common-components
  - /Users/simen/repos/monorepo/packages/folio-common-utils
info  - Using external babel configuration from /Users/simen/repos/monorepo/packages/om-folio/.babelrc
next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "next/head", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "next/router", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "styled-jsx/server", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/spacing.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/shadows.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/old-typography.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/spacing.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/shadows.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/no-child-vertical-margins.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextLikeInput.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/focus-styling.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/global-styles.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Autosuggest/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/old-typography.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/no-child-vertical-margins.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/focus-styling.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextLikeInput.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/global-styles.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Autosuggest/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/formatters.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/fnum.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/luhn.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/headings/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/utils.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/utils.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/invariant.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/ProgressBar/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/SignupForm/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/validators.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Select/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/VisuallyHidden/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/invariant.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/validators.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Spinner/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/track.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/focus-styling.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Divider/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/FileUploadArea/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Textarea/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/invariant.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/old-typography.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/headings/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/ProgressBar/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/SignupForm/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Select/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Spinner/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/VisuallyHidden/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Divider/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/FileUploadArea/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Switch/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Textarea/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/RadioButtonRow/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/RadioButton/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/AccountNumberInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TelephoneNumberInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/BirthNumberInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/NumberValueInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Label/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Logo/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/ErrorMessage/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Checkbox/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/numbers.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/org-number.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/normalizers.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/focus-styling.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Switch/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/date-fn-config.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/helpers.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/norwegian-pnum.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/RadioButton/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/RadioButtonRow/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/BirthNumberInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TelephoneNumberInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Logo/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/AccountNumberInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/ErrorMessage/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Label/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/NumberValueInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Checkbox/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/ArrowIcons.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/Button.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextLikeInput.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/mod11.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/luhn.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/mod11.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/normalizers.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/norwegian-pnum.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/PlainButton.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/TextButton.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/XIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/VisuallyHidden/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextLikeInput.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/track.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/org-number.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/numbers.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/normalizers.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/normalizers.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/mod11.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/mod11.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/luhn.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/luhn.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/fnum.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/formatters.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextLikeInput.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/ErrorMessage/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/norwegian-pnum.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/old-typography.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextLikeInput.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextLikeInput.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/VisuallyHidden/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextLikeInput.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/RadioButton/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/ErrorMessage/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/spacing.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Spinner/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/spacing.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Spinner/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/mod11.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/utils.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/TextButton.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/Button.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/PlainButton.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/ArrowIcons.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/XIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/TextInputContainer.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/TextInputContainer.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/spacing.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/spacing.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Spinner/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextInput/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/RadioButton/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/generated/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/invariant.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/norwegian-pnum.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/sharedStyles.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/sharedStyles.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/sharedStyles.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/mod11.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/utils.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/date-fn-config.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/lib/helpers.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-utils/dist/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/spacing.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextLikeInput.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/fonts.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/spacing.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/Spinner/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/TextLikeInput.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/generated/index.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/sharedStyles.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/sharedStyles.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/buttons/sharedStyles.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/generated/PurposeGoodsIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/generated/PurposeSpecifyIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/AtIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/ZipIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/WarningIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/TrashIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/TagIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/TadaIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/RefreshIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/ProfileIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/PlusIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/PhoneIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/OwnerPersonIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/AtIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/CheckmarkIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/CheckHollowIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/CompanyIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/EditIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/FingerprintIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/InboxIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/InfoIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/LinkIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/MailIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/OfficeIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/OutboxIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/OwnerCompanyIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/PlusIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/OwnerPersonIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/PhoneIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/ProfileIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/RefreshIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/TagIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/TadaIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/TrashIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/ZipIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/WarningIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/generated/PurposeGoodsIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/generated/PurposeHoldingIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/generated/PurposeServicesIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/generated/PurposeSpecifyIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/OwnerCompanyIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/OfficeIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/OutboxIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/MailIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/LinkIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/InfoIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/InboxIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/FingerprintIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/CompanyIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/EditIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/CheckmarkIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/generated/CheckHollowIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/generated/PurposeServicesIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/generated/PurposeHoldingIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/BaseIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/BaseIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/BaseIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/icons/BaseIcon.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/BaseIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/BaseIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/BaseIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/BaseIllustration.js
next-transpile-modules - transpiled: /Users/simen/repos/monorepo/packages/folio-common-components/dist/components/illustrations/BaseIllustration.js
event - compiled successfully
```

</details>

Reply:
^ I'm getting the same thing about unrelated dependencies that I don't want next-transpile-modules to touch

```
[0] next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "react/jsx-dev-runtime", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
[0] next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "react/jsx-dev-runtime", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
[0] next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "next/router", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
[0] next-transpile-modules - DEPRECATED - fallbacking to previous module resolution system for module "styled-jsx/server", you can now just pass the name of the package to transpile and it will detect its real path without you having to pass a sub-module.
```

Reply:
#199

Reply:
#208

Reply:
To anyone facing those issues, please comment on #208 and fill the Issue template information, thank you 🙏